codes.json

```diff
@@ -629,21 +629,21 @@
   ,
   {
     "title" : "FBPIC",
-    "subtitle" : "TBD/",
-    "description" : "TBD/",
-    "application": [],
-    "compute": [],
-    "method": [],
+    "subtitle" : "Fourier-Bessel Particle-In-Cell",
+    "description" : "FBPIC is a specialized electromagnetic Particle-In-Cell code for simulations of laser-wakefield acceleration and plasma-wakefield acceleration, with close-to-cylindrical symmetry. The code can run on multi-CPU and multi-GPU architectures (NVIDIA only), and uses a cylindrical geometry to speed up simulations.",
+    "application": ["laser-plasma"],
+    "compute": ["mpi", "gpu"],
+    "method": ["pic"],
     "phenomena": [],
     "homepage" : "https://fbpic.github.io",
-    "institutions" : ["TBD/"],
-    "institution_link" : "TBD/",
-    "repository" : "TBD/",
-    "issue_tracker" : "TBD/",
-    "platform": [],
+    "institutions" : ["Lawrence Berkeley National Laboratory", "CFEL, Hamburg University"],
+    "institution_link" : "https://www.lbl.gov",
```

Comment:
The other link would be CFEL, but I have to first change the string to array in the schema of all entries :)
http://lux.cfel.de/